README.md

```diff
@@ -10,6 +10,7 @@ How to build
 
 ```
 % git submodule update --init --recursive
+% curl -L https://cpanmin.us | perl - --installdeps --notest .
```

Comment:
I think we might want to split this to two lines, one installing cpanminus, and another installing dependencies, because we'd expect the former to be run only once, while the other might be required to be run whenever the dependency changes.